### ISSUE TITLE
[FEATURE REQUEST] Option to hide (Homepage) in the Developer section

### ISSUE THREAD

**Issue by lllIIIlll** [tfedor/AugmentedSteam#1638]
### Current Behavior (if any)

(Homepage) appears behind the Developer and Publisher sections and cannot be hidden.

![1](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/125834307/757453fa-8598-48a6-85e5-d9684d4decaa)

### Desired Behavior

Add an option to hide the (Homepage) shown behind the Developer and Publisher sections.

### Motivation / Use Case for Changing the Behavior

Less clutter, and some users don't want these features, so more options are always better.

**Comment by candela97** [comment]
We try to introduce features in a way that doesn't require an option, because each one adds maintenance cost (translations, additional space in Options...). This feature is a clear improvement, doesn't fetch any data, doesn't clutter the page too much and has been added for quite some time. If more people complain we can consider an option, or perhaps use an icon instead of text?

**Comment by candela97** [comment]
Yeah it does get pretty long sometimes. Would it help if each name is shown on a single line?
